modules/ocf/manifests/serial_getty.pp

```diff
@@ -0,0 +1,8 @@
+class ocf::serial_getty {
+  service { 'serial-getty@ttyS0':
+    ensure   => running,
```

Comment:
...it can often be months between server reboots, though, and an emergency could potentially develop during this time which forces us to use the serial console...